pkgs/kde/third-party/krohnkite/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nodejs_22,
+  kwin,
+  kpackage,
+  p7zip,
+  git,
+}:
+buildNpmPackage rec {
+  pname = "krohnkite";
+  version = "0.9.7";
+
+  src = fetchFromGitHub {
+    owner = "anametologin";
+    repo = pname;
+    rev = version;
+    hash = "sha256-8A3zW5tK8jK9fSxYx28b8uXGsvxEoUYybU0GaMD2LNw=";
+  };
+
+  npmDepsHash = "sha256-My1goFEoZW9kFA3zb8xKPxAPXm6bypyq+ajPM8zVOHQ=";
+
+  dontWrapQtApps = true;
+
+  nodejs = nodejs_22;
+
+  nativeBuildInputs = [
+    kpackage
+    p7zip
+    kwin
+    git
```

Comment:
Why do we need 7z and git?

Reply:
7z is used in the makefile to create the installable kwinscript. Could be replaced with zip?
Git to add the git rev to the metadata, can be ommitted I think.

Reply:
zip would be a lot more unixy, so if that is easy to do I would prefer that.

Reply:
Yeh no problem, a sed will do it.